source/jormungandr/jormungandr/realtime_schedule/tests/sytral_test.py

```diff
@@ -175,6 +215,7 @@ def mock_missing_line_response():
             {
                 "direction_id": "3341",
                 "direction_name": "Piscine Chambéry",
+                "direction_type": "both",
```

Comment:
We don't use this field anywhere, no?
Should we use it to attach it to the right route when reading response from subservice?
Also, if we keep it in tests, is it a realistic value?

Reply:
I added this filed in all mock struct but it is not use, yes. 
It just to be conform with the new Sytral-Rt response version. It is mandatory now